src/features/Licenses/LicensesFilters/index.jsx

```diff
@@ -34,14 +34,11 @@ const LicensesFilters = ({ resetPagination }) => {
     const form = e.target;
     const formData = new FormData(form);
     const formJson = Object.fromEntries(formData.entries());
-    const urlParamsFilters = Object.entries(formJson)
-      .map(([key, value]) => `${key}=${encodeURIComponent(value)}`)
```

Comment:
@01001110J should it be encoded in that part of the code?

Reply:
As far I remember ( please confirm with @AuraAlba  ) we should send the info encoded to avoid to get unexpected responses. The request params should be encoded, but please confirm with Aura first to be sure.

Reply:
@01001110J I tested with a License name with special characters and it worked.

![image](https://github.com/user-attachments/assets/f333d181-b363-44b2-8093-d7fafe183ba0)